coderefs/alias_test.go

```diff
@@ -71,6 +72,14 @@ func Test_GenerateAliases(t *testing.T) {
 			},
 			want: map[string][]string{testFlagKey: slice("SOME_FLAG")},
 		},
+		{
+			name:  "file wildcard pattern",
+			flags: slice(testFlagKey, testWildFlagKey),
+			aliases: []o.Alias{
+				fileWildPattern(testFlagKey),
+			},
+			want: map[string][]string{testWildFlagKey: slice("WILD_FLAG"), testFlagKey: slice("SOME_FLAG")},
+		},
 		// TODO
 		// {
 		// 	name:    "command",
```

Comment:
Is this TODO still valid or safe to remove?

Reply:
I'm not sure, I'll leave it there for this PR just to keep it moving along.